tox.ini

```diff
@@ -562,7 +562,7 @@ deps =
     strawberry-latest: strawberry-graphql[fastapi,flask]
 
     # Tornado
-    tornado-v5: tornado~=5.0
+    tornado-v6.0: tornado~=6.0.0
```

Comment:
This line also looks redundant

Reply:
It isn't, it tests the lowest supported version (6.0.x), whereas the line below tests whatever the latest 6.x version is

Reply:
With this we should cover the lowest and highest supported 6 version

Reply:
Cool, sounds good in that case